api/openapi.yaml

```diff
@@ -0,0 +1,341 @@
+openapi: "3.0.0"
```

Comment:
While scrolling around for the review, I noticed this file is a little large for navigating. Not an action for this PR, just noting a thought  - wondering if openapi would support it if we broke the yaml into multiple files e.g. paths and components in different files. Again, not an action item for this PR but thinking down the line as we expand on this file!

Reply:
Yeah, that's something we could definitely look into, I think you can reference schemas at the very least, so could break the schemas into their own files

Reply:
Creating an issue to track this for the future